src/util.jl

```diff
@@ -5,9 +5,23 @@ import Base: RefValue, @pure, display, show, clamp, find, cumsum, cumprod, cummi
 export constant, select, get_last_error, err_to_string, sort_index
 export mean_weighted, var_weighted, set_array_indexer, set_seq_param_indexer
 
-function afgc(threshold = 4e9)
+global const af_threshold = Ref(4*1024*1024*1024)
```

Comment:
This is already in global scope, is it  necessary to declare it a `global const` vs just a  `const`?